azure-quantum/tests.live/Run.ps1

```diff
@@ -76,6 +76,10 @@ $logs = Join-Path $env:BUILD_ARTIFACTSTAGINGDIRECTORY "logs" "qdk-python.txt"
 
 # Copy unit tests without recordings and run Pytest
 Copy-Item -Path (Join-Path $PackageDir "tests" "unit" "*.py") -Destination $PSScriptRoot
+if ($PackageDir -contains "azure-quantum") {
```

Comment:
Looks good, but could you add a comment explaining what/why this is copying?

Reply:
I added a comment on what is copied.